jira.py

```diff
@@ -85,6 +102,59 @@ def download_issue(issue_key: str, fields):
     return result
 
 
+def save_user_photo(user_name: str, photo_urls):
+    if user_name == None or len(photo_urls) == 0:
+       print('{} has no avatars'.format(user_name))
+       return
+   
+    directory_path = 'user_image_dir'
+    if os.path.exists(directory_path) == False:
+        os.makedirs(directory_path, exist_ok=True)
+    path_to_avatar_file = '{}/{}.png'.format(directory_path, user_name)
+    if os.path.exists(path_to_avatar_file):
+       return
```

Comment:
For some reason, two of the `if`s in this block have three spaces of indentation instead of four.
```suggestion
    if user_name == None or len(photo_urls) == 0:
        print('{} has no avatars'.format(user_name))
        return
   
    directory_path = 'user_image_dir'
    if os.path.exists(directory_path) == False:
        os.makedirs(directory_path, exist_ok=True)
    path_to_avatar_file = '{}/{}.png'.format(directory_path, user_name)
    if os.path.exists(path_to_avatar_file):
        return
```